style.css

```diff
@@ -0,0 +1,39 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.container {
+  padding-top: 50px;
+  width: 100%;
+  height: 100%;
+  display: grid;
+  grid-template-columns: 20% 80%;
+  background-color: #eee;
+}
+
+.sideNav {
+  height: 100vh;
+  padding: 1rem;
+}
+
+.sideNav h1 {
+  text-transform: capitalize;
+}
+
+.sideNav ul {
+  list-style-type: none;
+  width: 100%;
+  overflow: scroll;
+}
+
+.sideNav ul li {

```

Comment:
Please use direct child selector like  .sideNav ul > li
instead of Camel Case. try to use dashes ( - ) or Underscores ( _ )